index.html

```diff
@@ -134,14 +134,23 @@ <h2 class="text-center">Calculate what your portion of the rent should be.</h2>
     </section>
 </main>
 
-<footer class="flex-column flex-vert-bottom flex-horiz-center   text-uppercase">
-    <aside class="text-center">
+<footer class="flex-column flex-vert-bottom flex-horiz-center   text-uppercase text-center">
+    <aside class="text-normal">
+        Like this project and want to keep it rolling?
+        Support us monthly on <a href="https://www.patreon.com/BlueHuskyStudios" target="_blank">Patreon</a>,
+        or just once via the lead developer's <a href="https://paypal.me/BenLeggiero" target="_blank">PayPal</a>.
```

Comment:
I don't like doing this, but it seems so long as Blue Husky Studios is just a brand and not a registered corporation, this is our only option.